lib/widgets/screens/profile/profile_basic_info.dart

```diff
@@ -0,0 +1,231 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+import 'package:mm_flutter_app/utilities/scaffold_utils/report_or_block_menu_button.dart';
+import 'package:mm_flutter_app/widgets/atoms/profile_chip.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+const double _inlineIconSize = 16.0;
+
+Future<void> _launchUrl(Uri url) async {
+  if (!await launchUrl(url)) {
```

Comment:
Definitively out of scope for this PR, but launching a URL like this basically let's LinkedIn take over our app. The only way the user can go back to our main app is by pressing back enough times to pop all the pages in LinkedIn, or by closing and reopening the app. We might need to consult UX on how we should handle this: opening externally vs. opening inside the app (with perhaps a "Close" button to go back immediately).

I will add this to their to-do list.

Reply:
Makes sense!